PCLConfig.cmake.in

```diff
@@ -211,22 +211,18 @@ macro(find_rssdk)
   find_package(RSSDK)
 endmacro(find_rssdk)
 
-macro(find_realsense2)
+macro(find_rssdk2)
+
   if(PCL_ALL_IN_ONE_INSTALLER)
```

Comment:
@UnaNancyOwen -- I feel like lines 216-221 are no longer necessary for the all-in-one installer, but can you please verify?

Reply:
It's necessary yet.
In case of PCL All-in-one Installer, <code>realsense2_DIR</code> needs to refer to <code>PCL_ROOT/3rdParty/librealsense2</code>.

Reply:
resolved to librealsense2 in both places